Upgrade Celo Dependencies

aaronmgdr opened the pull request:
### Description

celo package upgrades have some important upgrades

also its best to upgrade web3 at same time as contractkit.

Remove test for demurage/inflation (this feature was removed from StableToken Contract)

### Other changes

yarn install automatically added packageManager field. 

### Tested

aaronmgdr committed:
keeping up with the jones

soloseng reviewed (commented):
Seems to be a conflict with the web3 dependency, that is causing build to fail. Might be worth also updating web3 dep

aaronmgdr committed:
upgrade to lastest contractkit -- for better l2 compatability

remove demurage / inflation test on Stable transfer e2e as that no longer exists

aaronmgdr commented:
I dont see how this could have caused anvil devchain test to break
